datumaro/plugins/icdar_format/extractor.py

```diff
@@ -89,8 +89,10 @@ def _load_localization_items(self):
 
         image_dir = osp.join(self._path, IcdarPath.IMAGES_DIR)
         if osp.isdir(image_dir):
-            images = { osp.splitext(osp.relpath(p, image_dir))[0]: p
-                for p in find_images(image_dir, recursive=True) }
+            images = {
+                osp.splitext(osp.relpath(p, image_dir))[0].replace('\\', '/'): p
```

Comment:
Maybe it's time to update `find_images()` itself? It has the same pattern of use everywhere.

Reply:
Not everywhere. Many formats don't build a lookup table from the `find_images` results, and the path separator doesn't become an issue for them.